re.sonny.OhMySVG.yaml

```diff
@@ -0,0 +1,18 @@
+app-id: re.sonny.OhMySVG
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+command: re.sonny.OhMySVG
+finish-args:
+  - "--share=ipc"
```

Comment:
```suggestion
  - "--share=ipc"
  - "--device=dri"
```

Reply:
why?

Reply:
```
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: MESA-LOADER: failed to open amdgpu: /usr/lib/x86_64-linux-gnu/GL/default/lib/dri/amdgpu_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/GL/default/lib/dri)
libGL error: failed to load driver: amdgpu
libGL error: failed to open /dev/dri/card0: No such file or directory
libGL error: failed to load driver: radeonsi
```

It probably doesn't matter much in the context of your app but would be nice to fix.

Reply:
:+1: